### ISSUE TITLE
[UPDATE] skin.pellucid 2.1.0

### ISSUE THREAD

**Issue by chrisbevan** [xbmc/repo-skins#208]
- Add-on name: Pellucid
- Add-on ID: skin.pellucid
- Version number: 2.1.0
- Kodi/repository version: Leia
- URL: https://github.com/chrisbevan/skin.pellucid
- Revision/tag: a167dd70e3a9b1b7f6080bba8df3fb1b56329afb
- Branch: master

### Description of changes:
- Various minor updates

### Checklist:
- [x] My code follows the [add-on rules](http://kodi.wiki/view/Add-on_rules) and [piracy stance](http://kodi.wiki/view/Official:Forum_rules#Piracy_Policy) of this project.
- [x] I have read the [CONTRIBUTING](https://github.com/xbmc/repo-skins/blob/master/CONTRIBUTING.md) document
- [x] Each add-on submission should be a single commit with using the following style: [plugin.video.foo] v1.0.0

**Comment by Rechi** [comment]
```
INFO: Checking add-on skin.pellucid
INFO: Created by theDeadman
INFO: Addon id matches folder name
INFO: skin.pellucid addon also exists in krypton branch but with version: 1.3.3
INFO: Valid XML file found
WARN: http://forum.kodi.tv/forumdisplay.php?fid=267 redirects to https://forum.kodi.tv/forumdisplay.php?fid=267
ERROR: ./skin.pellucid/extras/homeBackgrounds/DSC03863.jpg is marked as stand-alone executable
ERROR: ./skin.pellucid/extras/homeBackgrounds/DSC03910.jpg is marked as stand-alone executable
ERROR: ./skin.pellucid/extras/homeBackgrounds/DSC04160.jpg is marked as stand-alone executable
ERROR: ./skin.pellucid/fonts/roboto_slab/bold.ttf is marked as stand-alone executable
ERROR: ./skin.pellucid/fonts/roboto_slab/LICENSE.txt is marked as stand-alone executable
ERROR: ./skin.pellucid/fonts/roboto_slab/light.ttf is marked as stand-alone executable
ERROR: ./skin.pellucid/fonts/roboto_slab/regular.ttf is marked as stand-alone executable
ERROR: ./skin.pellucid/fonts/roboto_slab/thin.ttf is marked as stand-alone executable
INFO: Image icon exists
INFO: Icon dimensions are fine 512x512
INFO: Image fanart exists
INFO: Fanart dimensions are fine 1280x720
INFO: Image screenshot exists
INFO: Image screenshot exists
INFO: Image screenshot exists
INFO: Image screenshot exists
INFO: Image screenshot exists
INFO: Image screenshot exists
ERROR: Using the old language directory structure, please move to the new one.
ERROR: Invalid PO file .\skin.pellucid\language\Spanish\strings.po: Syntax error on line 1
ERROR: We found 10 problems and 1 warnings, please check the logfile.
```

**Comment by ronie** [comment]
thanx for the update!
no comments apart from a few cosmetic things:

**addon.xml:**
please change lang="en" to lang="en_GB"
please change the forum link to https://

**language/Spanish/strings.po**
please add a space after the `#` when using inline comments, e.g.
`# This is the correct way to add a comment`
`#This is not correct`

please remove the executable bit on the files mentioned in the previous reply.

and finally, rename your language folers, e.g.
`English` -> `resource.language.en_gb`

**Comment by chrisbevan** [comment]
Thanks Ronie, fixed in dc1ad25832a1f9b0461a4f1e796c7f3fbc30351f

Cheers,
Chris.

**Comment by ronie** [comment]
updated, thx!
https://github.com/xbmc/repo-skins/commit/5d652c7972962b19d4a145a59792d6e16d6861b8
